docs/tools/developer-tools/quickstart/network-modes.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Network Modes
+description: Quickstart supports the network modes local, testnet, futurenet and pubnet
+sidebar_label: Network Modes
+sidebar_position: 20
+---
+
+# Network Modes
+
+Quickstart can operate in different network modes, and you need to decide which mode is suitable for your purpose before starting the container. The four available network modes are **_local_**, **_testnet_**, **_futurenet_** and **_pubnet_** (mainnet).
```

Comment:
very loosely held opinion: these four network modes you have might be better suited to using the backticks, rather than both bold/italic?